docs/sources/panels-visualizations/visualizations/bar-gauge/index.md

```diff
@@ -67,6 +67,22 @@ Choose a display mode.
 - **Retro LCD -** The gauge is split into small cells that are lit or unlit.
 - **Basic -** Single color based on the matching threshold.
 
+### Value display
+
+Choose a value display mode.
+
+- **Value color -** Value color is determined by value.
+- **Text color -** Value color is default text color.
+- **Hidden -** Values are hidden.
+
+### Value placement
```

Comment:
Question: I might have misunderstood this feature but I feel that the naming of this option is a bit confusing. I thought it was the field name that you configured the placement for and not the value?

Reply:
ah 🤦‍♂️ you are totally right - not sure where my head was at when naming this haha, it is should be called `Field label placement`